corehq/apps/couch_sql_migration/diff.py

```diff
@@ -306,6 +307,15 @@ def has_date_values(old_obj, new_obj, rule, diff):
     return _both_dates(diff.old_value, diff.new_value)
 
 
+def sql_number_has_leading_zero(old_obj, new_obj, rule, diff):
```

Comment:
huh, any idea where the leading 0 comes from? Is this on case properties? Maybe you could add a comment here as to why this might happen and why it seems okay.

Reply:
The number is formatted like that in the XML, so I'm assuming it was a user-entry anomaly. The couch form processor stripped off the leading zero somehow (I'm not sure how or why). Here's an example XML snippet:
```xml
<n0:affiliation_num_mother>00.3</n0:affiliation_num_mother>
```

I'll add a comment.